src/lib/server/firebaseUtils.ts

```diff
@@ -141,3 +143,10 @@ export const writePath = async (path: string, data: any) => db.ref(path).set(dat
 export const pushPath = async (path: string, data: any) => db.ref(path).push(data);
 
 export const deletePath = async (path: string) => db.ref(path).remove();
+
+export const modifyPlaylistsResponse = (playlists: Database.Playlist[]) => {
+  return Object.values(playlists).map((playlist) => ({
+    ...playlist,
+    words: playlist.words.map((word) => word.map((letter) => (letter === false ? null : letter))),
+  }));
+};
```

Comment:
I don't think this should be in this file since it doesn't actually touch Firebase

Reply:
true i can move it to just utils!

Reply:
jk i just moved it into the actual server file since it was only being used there anyways